### DISCUSSION TITLE
How to offline install different node.js version on nwm

### DISCUSSION THREAD

**Discussion by asarper** [coreybutler/nvm-windows#974] [General]
Hello,

We use offline systems so how can I install other node.js version on nwm  ? 

Thank you

**Comment by coreybutler** [comment]
It's not possible unless your organization is running a Node.js mirror on your network. Somehow, the files have to be transferred to your computer. If there is no connection, then there's no way to do that short of using physical media.

If you are able to connect occasionally, then you could install multiple versions when you are connected, then just `nvm use <version>` when you are offline.

**Reply by asarper** [reply]
Hello,
Thank you very much for your answer. Can I run the different version node.js setup file that I transferred to the closed environment with nvm?
